### PR TITLE
✨ Add lucky compass, lucky rabbit's foot, totem of undying, war hog's burning rage, and all armor stands to loot tables

### LINKED ISSUE #36: Move some items to loot tables
These items should be quick to implement and relatively simple:
- [x] Aurelia's Lucky Compass
- [x] Totem of Undying
- [x] War Hog's Burning Rage
- [x] Lucky Rabbit's Foot
- [x] Armor Stands with Arms (None, Baby, Bashful, Can-Can, Gallant, Greyskull, Skeleton, Zombie; each should have their own loot table)

Optionally use [misode's Loot Table Generator](https://misode.github.io/loot-table/) to create the new loot tables.

### PR DESCRIPTION
**Pull request by shrinerh**
Fixes #36

### CODE REVIEW

File: src/foxcraft_dungeon_loot/items.mc [hunk 1]
```diff
@@ -116,13 +116,13 @@ dir shield_of_the_juggernaut {
 
 dir aurelias_lucky_compass {
     function give {
-        give @s compass{DungeonLootId:18,AttributeModifiers:[{AttributeName:"generic.luck",Amount:2,Slot:offhand,Name:"generic.luck",UUID:[I;-122527,353760,12513,-707520]},{AttributeName:"generic.luck",Amount:2,Slot:mainhand,Name:"generic.luck",UUID:[I;-122527,353860,12513,-707720]}],display:{Name:'[{"text":"Aurelia\'s Lucky Compass","italic":false,"color":"aqua"}]',Lore:['[{"text":"Find yourself some nice booty.","italic":true}]','[{"text":"Rarity: ","italic":true},{"text":"Rare","color":"aqua","italic":true}]']},Enchantments:[{id:mending,lvl:1}],HideFlags:1} 1
+        loot give @s loot foxcraft_dungeon_loot:items/rare/aurelias_lucky_compass
```
Comment (satyrnidae) [inline src/foxcraft_dungeon_loot/items.mc]: replace command w/ macro notation:
`macro give_as_loot <rarity>\<table name>`

**Review by satyrnidae — CHANGES_REQUESTED** [review]
macros pls

**Review by satyrnidae — DISMISSED** [review]
neato

File: src/foxcraft_dungeon_loot/items.mc (hunk 1 above)
Comment (shrinerh) [reply src/foxcraft_dungeon_loot/items.mc]: done

**Review by satyrnidae — APPROVED** [review]
okey dokey

**Pull request merged**